src/governance/VoltVetoGovernor.sol

```diff
@@ -0,0 +1,398 @@
+        emit QuorumUpdated(_quorum, newQuorum);
+        _quorum = newQuorum;
+    }
+
+    /// ------------------------------------------------------------------------
+    /// Timelock Management
+    /// ------------------------------------------------------------------------
+
+    /**
+     * @dev Emitted when the timelock controller used for proposal execution is modified.
+     */
+    event TimelockChange(address oldTimelock, address newTimelock);
+
+    /// @notice the timelock linked to this veto governor
+    TimelockController private _timelock;
+
+    /// @notice mapping of proposalId (in this Governor) to timelockId (action ID in
+    /// the timelock linked to this governor).
+    mapping(uint256 => bytes32) private _timelockIds;
+
+    /**
+     * @dev Public accessor to check the address of the timelock
+     */
+    function timelock() public view returns (address) {
+        return address(_timelock);
+    }
+
+    /// @notice Set the timelock this veto governor can cancel from.
+    function updateTimelock(
+        address newTimelock
+    ) external onlyVoltRole(VoltRoles.GOVERNOR) {
+        _updateTimelock(newTimelock);
+    }
+
+    function _updateTimelock(address newTimelock) private {
+        emit TimelockChange(address(_timelock), newTimelock);
+        _timelock = TimelockController(payable(newTimelock));
+    }
+
+    /// ------------------------------------------------------------------------
+    /// Vote counting
+    /// ------------------------------------------------------------------------
+
+    /**
+     * @dev Supported vote types. Matches Governor Bravo ordering.
+     */
+    enum VoteType {
+        Against,
+        For,
+        Abstain
+    }
+
+    struct ProposalVote {
+        uint256 againstVotes;
+        uint256 forVotes;
+        uint256 abstainVotes;
+        mapping(address => bool) hasVoted;
+    }
+
+    mapping(uint256 => ProposalVote) private _proposalVotes;
+
+    /**
+     * @dev See {IGovernor-COUNTING_MODE}.
+     */
+    // solhint-disable-next-line func-name-mixedcase
+    function COUNTING_MODE()
+        public
+        pure
+        virtual
+        override
+        returns (string memory)
+    {
+        return "support=bravo&quorum=against";
+    }
+
+    /**
+     * @dev See {IGovernor-hasVoted}.
+     */
+    function hasVoted(
+        uint256 proposalId,
+        address account
+    ) public view virtual override returns (bool) {
+        return _proposalVotes[proposalId].hasVoted[account];
+    }
+
+    /**
+     * @dev Accessor to the internal vote counts.
+     */
+    function proposalVotes(
+        uint256 proposalId
+    )
+        public
+        view
+        virtual
+        returns (uint256 againstVotes, uint256 forVotes, uint256 abstainVotes)
+    {
+        ProposalVote storage proposalvote = _proposalVotes[proposalId];
+        return (
+            proposalvote.againstVotes,
+            proposalvote.forVotes,
```

Comment:
This terminology is confusing, because forVotes are voting for a cancellation, and againstVotes are voting against cancellation. I'd recommend either changing the terminology to `cancelVotes` and `proceedVotes` or adding comments that describes the variable names on each function.

Reply:
Ok, it's not confusing the for the reason I pointed out. forVotes will always be 0, because no for votes are allowed, so there should be a comment here explaining this, or instead of reading from storage for `forVotes`, just hardcode it to return 0 because no forVotes are allowed to be cast.